rent_golem/backend/rent_golem/clusters/tests.py

```diff
@@ -1,3 +1,60 @@
-from django.test import TestCase
+from django.shortcuts import reverse
 
-# Create your tests here.
+from rest_framework.test import APIClient, APITestCase, APIRequestFactory
+from rest_framework import status
+
+
+class ClustersViewSetTest(APITestCase):
+    def setUp(self) -> None:
+        self.factory = APIRequestFactory()
+        self.client = APIClient()
+
+    def create_dummy_db_object(self):
+        json_data = {
+            "uuid": "dcec5482-59ef-4c0e-9ea4-4e5451c3cbda",
+            "package_type": "automatic",
+            "status": "Starting",
+            "additional_params": {},
+            "size": 5,
+        }
+        return self.client.post(reverse('clusters-list'), data=json_data, format="json"), json_data
```

Comment:
When preparing data for tests you don't have to use API. You can just call `Cluster.objects.create(...)`.